app/services/random_act_service.rb

```diff
@@ -0,0 +1,14 @@
+class RandomActService
+  def self.conn
+    Faraday.new(url: "https://localhost:3000")
```

Comment:
This looks great!!! As a refactor we can consider taking the end of the URL from line 12 and connecting it to the URL in line 3. On the BE this API call will never change and so what it delivers to the FE will probably also never change. : )